### PR TITLE
feat(dashboard): add backend support to automated analysis card embeded in dashboard on -web

### LINKED ISSUE #1242: Automated Analysis Report information, but only orange/red scores

### PR DESCRIPTION
**Pull request by maxcao13**
Fixes #1242

### CODE REVIEW

**Review by andrewazores — DISMISSED** [review]
LGTM

**Review by tthvo — DISMISSED** [review]
Looks great to me :D

**Comment by andrewazores** [comment]
@maxcao13 rebase please, GH says it'll have conflicts so it won't let me do it automatically

File: src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java [hunk 1]
```diff
@@ -67,29 +67,44 @@
...
                         .softValues()
-                        .build((k) -> getReport(k));
+                        .build((k) -> getReport(k, "", true));
+        this.jsonCache =
+                Caffeine.newBuilder()
+                        .scheduler(Scheduler.systemScheduler())
+                        .expireAfterWrite(cacheExpirySeconds, TimeUnit.SECONDS)
+                        .refreshAfterWrite(cacheRefreshSeconds, TimeUnit.SECONDS)
+                        .softValues()
+                        .build((k) -> getReport(k, "", false));
```
Comment (andrewazores) [inline src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java:107]: Maybe worth adding an overload for `getReport(k, false)` that passes this empty filter implicitly. And perhaps a named constant `NO_FILTERS = ""` would be useful for readability too?

**Review by andrewazores — APPROVED** [review]

**Pull request merged**